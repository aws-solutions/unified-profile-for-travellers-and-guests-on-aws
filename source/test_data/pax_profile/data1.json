{
  "modelVersion": "1.0",
  "id": "0049074682",
  "lastUpdatedOn": "2022-01-24T04:42:27.919547161Z",
  "createdOn": "2021-09-24T04:42:27.919547161Z",
  "lastUpdatedBy": "Thaddeus Gusikowski",
  "createdBy": "Garrett Lemke",
  "emails": [
    {
      "type": "",
      "address": "travisdurgan@sanford.name",
      "primary": false
    },
    {
      "type": "",
      "address": "mathewcorwin@walter.info",
      "primary": true
    }
  ],
  "phones": [
    {
      "type": "business",
      "number": "465.725.7891",
      "primary": false,
      "countryCode": 90
    },
    {
      "type": "business",
      "number": "1389184526",
      "primary": true,
      "countryCode": 85
    }
  ],
  "addresses": [],
  "honorific": "Sir",
  "firstName": "Herman",
  "middleName": "Aglae",
  "lastName": "Bergnaum",
  "gender": "other",
  "pronoun": "they",
  "dateOfBirth": "1960-12-29",
  "language": {
    "code": "ur",
    "name": ""
  },
  "nationality": {
    "code": "SZ",
    "name": ""
  },
  "jobTitle": "Technician",
  "parentCompany": "SnapSense",
  "loyaltyPrograms": [
    {
      "id": "2697555871",
      "programName": "elite",
      "miles": 292878.0,
      "milesToNextLevel": 6671.0,
      "level": "platinium",
      "joined": "2021-09-24T04:42:27.919547161Z"
    }
  ],
  "identityProofs": null
}