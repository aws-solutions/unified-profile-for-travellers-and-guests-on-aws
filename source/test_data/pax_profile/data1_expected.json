{"air_profile_recs": [{"object_type": "pax_profile", "model_version": "1.0", "id": "0197775441", "last_updated": "2021-10-16T00:16:33.420688138Z", "created_on": "2021-08-16T00:16:33.420688138Z", "last_updated_by": "Newton Pacocha", "created_by": "Warren Dickens", "honorific": "Mx", "first_name": "Kian", "middle_name": "Danyka", "last_name": "Dietrich", "gender": "other", "pronoun": "they", "date_of_birth": "1939-01-18", "job_title": "Technician", "company": "Copyright Clearance Center", "nationality_code": "VU", "nationality_name": "", "language_code": "rw", "language_name": "", "email_business": "", "email": "gudrunprice@swift.net", "email_type": "", "phone": "", "phone_home": "", "phone_mobile": "", "phone_business": "", "traveller_id": "0197775441", "last_updated_partition": "2021/10/16/00/16/33/420688138Z", "address_line1": "", "address_line2": "", "address_line3": "", "address_line4": "", "address_city": "", "address_state_province": "", "address_postal_code": "", "address_country": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_postal_code": "", "address_billing_country": "", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": "shipping"}], "common_email_recs": [{"model_version": "1.0", "object_type": "email_history", "last_updated": "2021-10-16T00:16:33.420688138Z", "last_updated_by": "Newton Pacocha", "address": "gudrunprice@swift.net", "type": "", "traveller_id": "0197775441", "last_updated_partition": "2021/10/16/00/16/33/420688138Z"}, {"model_version": "1.0", "object_type": "email_history", "last_updated": "2021-10-16T00:16:33.420688138Z", "last_updated_by": "Newton Pacocha", "address": "mireyawillms@kuphal.biz", "type": "", "traveller_id": "0197775441", "last_updated_partition": "2021/10/16/00/16/33/420688138Z"}], "common_phone_recs": [], "air_loyalty_recs": [{"model_version": "1.0", "object_type": "air_loyalty", "last_updated": "2021-10-16T00:16:33.420688138Z", "last_updated_by": "Newton Pacocha", "id": "1726659371", "program_name": "elite", "miles": 314504.0, "miles_to_next_level": 4831.0, "level": "silver", "joined": "2021-08-16T00:16:33.420688138Z", "traveller_id": "0197775441", "last_updated_partition": "2021/10/16/00/16/33/420688138Z"}, {"model_version": "1.0", "object_type": "air_loyalty", "last_updated": "2021-10-16T00:16:33.420688138Z", "last_updated_by": "Newton Pacocha", "id": "5978089185", "program_name": "club", "miles": 586674.0, "miles_to_next_level": 2109.0, "level": "bronze", "joined": "2021-08-16T00:16:33.420688138Z", "traveller_id": "0197775441", "last_updated_partition": "2021/10/16/00/16/33/420688138Z"}]}